reledmac.dtx

```diff
@@ -2699,11 +2743,29 @@
 % For  critical endnotes NOT paragraphed you can define an indentation with \protect\cs{Xendhangindent}\oarg{s}\marg{l}, which will be applied in the second line of notes. It can help to make a distinction between a new note and a break in a note.
 %
 %
-% \subsection{Arbitrary code at the beginning of notes}
+% \subsection{Arbitrary code}
+
+% \subsubsection{Arbitrary code around line number}
+% \DescribeMacro{\Xendbhooklinenumber}
+% \cs{Xendbhooklinenumber}\oarg{s}\marg{code} is used to execute code before line numbers in endnotes. The code is executed before the \cs{Xendbeforelinenumber} space and before the \cs{Xendnotenumfont} font setting.
+%
+% \DescribeMacro{\Xendahooklinenumber}
+% \cs{Xendahooklinenumber}\oarg{s}\marg{code} is used to execute code after line number in endnotes.
+% The code is executed after the \cs{Xendafternumber} space.
+%
+%
+%
+% \DescribeMacro{\Xendbhookinplaceofnumber}
+% \cs{Xendbhookinplaceofnumber}\oarg{s}\marg{code} is used to execute code before space or symbol which replace line number in endnotes. The code is executed before the \cs{Xendbeforesymlinenum} space and before the \cs{Xendnotenumfont} font setting.
```

Comment:
"before space or symbol which replace line number": I can't tell whether this is "before (space or symbol which replace line number)" or "before space or (symbol which replace line number)". In the first case, write "before a space or symbol which replaces the line number"; in the second case, write "before a space, or a symbol which replaces the line number".

Reply:
I tooks time to understand your point. It was about the relationship between "replace line  number" and space / symbol, right? And not, as I thought in first reading, concerning the relationship between "before" and space/symbol.

So I think the first solution is good. Some time, line number is replace by space OR by a symbol.

Reply:
Sorry I wasn't clear; I was trying to indicate binding with the parens. I agree, from your description it's the first case, so my first solution should be used!

Reply:
Cool! Done.